apps/web/ee/pages/workflows/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from "react";
+
+import checkLicense from "@calcom/ee/server/checkLicense";
+import { Alert } from "@calcom/ui/Alert";
+import LicenseRequired from "@ee/components/LicenseRequired";
+import { NewWorkflowButton } from "@ee/components/workflows/NewWorkflowButton";
+import WorkflowList from "@ee/components/workflows/WorkflowListPage";
+
+import { useLocale } from "@lib/hooks/useLocale";
+import useMeQuery from "@lib/hooks/useMeQuery";
+import { trpc } from "@lib/trpc";
+
+import Loader from "@components/Loader";
+import Shell from "@components/Shell";
+
+function WorkflowsPage() {
+  const { t } = useLocale();
+
+  const [isValidLicense, setIsValidLicense] = useState(false);
+
+  useEffect(() => {
+    const isLicenseValid = async () => {
+      const isValid: boolean = await checkLicense(process.env.CALCOM_LICENSE_KEY || "");
+      setIsValidLicense(isValid);
+    };
+    isLicenseValid();
+  });
```

Comment:
```suggestion
```

You can just use hasValidLicense from useSession:

```
const { data} = useSession();
  data?.hasValidLicense
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix